ClavaWeaver/src/pt/up/fe/specs/clava/weaver/CxxWeaver.java

```diff
@@ -2099,5 +2098,13 @@ public AstMethods getAstMethods() {
         return new ClavaAstMethods(this, ClavaNode.class, node -> CxxJoinpoints.create(node),
                 node -> ClavaCommonLanguage.getJoinPointName(node), node -> node.getScopeChildren());
     }
+    
+    @Override
+    public List<LaraResourceProvider> getNpmResources() {
+        return Stream.concat(
+                super.getNpmResources().stream(),
+                Arrays.asList(ClavaApiJsResource.values()).stream())
+            .toList();
+    }
 
```

Comment:
Verify if in future PRs (or staging) you still need to explicitly call super.getNpmResources()